atef/widgets/config.py

```diff
@@ -902,8 +902,10 @@ def handle_locking(self, locked: bool):
             self.delete_button.setEnabled(False)
         else:
             self.desc_edit.setFrameShape(self.desc_edit.StyledPanel)
+            color = self.palette().color(QPalette.ColorRole.Base)
             self.setStyleSheet(
-                "QLineEdit, QPlainTextEdit { background: white }"
+                f"QLineEdit, QPlainTextEdit {{ background: rgba({color.red()},"
+                f"{color.green()}, {color.blue()}) }}"
```

Comment:
```suggestion
                f"{color.green()}, {color.blue()}, {color.alpha()}) }}"
```

Reply:
I'm going to pull this suggestion in and merge